{"uid":"cc7c7fc9109bc6a5","name":"An authenticated Service Provider cannot send an RTP to an activated debtor","fullName":"A Creditor Service Provider sends an RTP through API: An authenticated Service Provider cannot send an RTP to an activated debtor","historyId":"6405207ec163b1b347ffd9712145ef9a","time":{"start":1759476898407,"stop":1759476898913,"duration":506},"status":"broken","statusMessage":"requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: https://api-mcshared.uat.cstar.pagopa.it/auth/token\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"bdd-tests/steps/auth_steps.py\", line 17, in given_ec_on_page\n    access_token = get_valid_access_token(client_id=client_id,\n                                          client_secret=client_secret,\n                                          access_token_function=get_access_token)\n  File \"/home/runner/work/rtp-platform-qa/rtp-platform-qa/api/auth.py\", line 8, in get_valid_access_token\n    token_response.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/requests/models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: https://api-mcshared.uat.cstar.pagopa.it/auth/token\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"bdd-tests/steps/auth_steps.py\", line 17, in given_ec_on_page\n    access_token = get_valid_access_token(client_id=client_id,\n                                          client_secret=client_secret,\n                                          access_token_function=get_access_token)\n  File \"/home/runner/work/rtp-platform-qa/rtp-platform-qa/api/auth.py\", line 8, in get_valid_access_token\n    token_response.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/requests/models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","steps":[{"name":"Given the debtor A has fiscal code random","time":{"start":1759476898408,"stop":1759476898408,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And the debtor Service Provider is authenticated","time":{"start":1759476898408,"stop":1759476898912,"duration":504},"status":"broken","statusMessage":"requests.exceptions.HTTPError: 502 Server Error: Bad Gateway for url: https://api-mcshared.uat.cstar.pagopa.it/auth/token\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"bdd-tests/steps/auth_steps.py\", line 17, in given_ec_on_page\n    access_token = get_valid_access_token(client_id=client_id,\n                                          client_secret=client_secret,\n                                          access_token_function=get_access_token)\n  File \"/home/runner/work/rtp-platform-qa/rtp-platform-qa/api/auth.py\", line 8, in get_valid_access_token\n    token_response.raise_for_status()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.7/x64/lib/python3.13/site-packages/requests/models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Given the creditor Service Provider is authenticated","time":{"start":1759476898913,"stop":1759476898913,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And the debtor Service Provider activated the debtor A","time":{"start":1759476898913,"stop":1759476898913,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When the creditor Service Provider is unauthenticated","time":{"start":1759476898913,"stop":1759476898913,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And the creditor Service Provider sends an RTP to the debtor A","time":{"start":1759476898913,"stop":1759476898913,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then the RTP is not created because the Service Provider is not authenticated","time":{"start":1759476898913,"stop":1759476898913,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":7,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"authentication"},{"name":"tag","value":"send"},{"name":"tag","value":"unhappy_path"},{"name":"feature","value":"A Creditor Service Provider sends an RTP through API"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":4,"unknown":0,"total":5},"items":[{"uid":"df0da4155e497b3e","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/551//#testresult/df0da4155e497b3e","status":"passed","time":{"start":1759390900128,"stop":1759390902179,"duration":2051}},{"uid":"cd8176cb3a77012d","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/550//#testresult/cd8176cb3a77012d","status":"passed","time":{"start":1759316306622,"stop":1759316309194,"duration":2572}},{"uid":"d2f289ba4de1ad92","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/549//#testresult/d2f289ba4de1ad92","status":"passed","time":{"start":1759303869644,"stop":1759303872738,"duration":3094}},{"uid":"ac9b0aa0e67eac1e","reportUrl":"https://pagopa.github.io/rtp-platform-qa/bdd/548//#testresult/ac9b0aa0e67eac1e","status":"passed","time":{"start":1759217446493,"stop":1759217448440,"duration":1947}}]},"tags":["send","unhappy_path","authentication"]},"source":"cc7c7fc9109bc6a5.json","parameterValues":[]}